app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,17 @@
+class SessionsController < ApplicationController
+  def create
+    user = User.find_by email: params[:session][:email].downcase
+    if user&.authenticate params[:session][:password]
+      log_in user
+      redirect_to user
+    else
+      flash.now[:danger] = t ".flash"
+      render :new
+    end
+  end
+
+  def destroy
+    log_out
+    redirect_to root_url
```

Comment:
```suggestion
    redirect_to root_path
```
nên dùng đường dẫn tương đối trong project thay vì đường dẫn tuyệt đối nhé